src/frontend/src/common/components/library/data_subview/components/MoreActionMenu/index.tsx

```diff
@@ -74,12 +73,12 @@ const MoreActionsMenu = ({
         onClose={handleClose}
         getContentAnchorEl={null}
       >
-        {usesFeatureFlag(FEATURE_FLAGS.editSamples) && (
-          <MenuItem onClick={handleEditSamples}>
-            <StyledEditIcon />
-            <StyledText>Edit Samples</StyledText>
-          </MenuItem>
-        )}
+        {/* {usesFeatureFlag(FEATURE_FLAGS.editSamples) && ( */}
```

Comment:
Do you actually want to turn off this feature flag?

Reply:
no that shouldn't have gotten in here, thanks removing now